samples/AntiforgerySample/project.json

```diff
@@ -30,5 +30,14 @@
   "content": [
     "wwwroot",
```

Comment:
is there any content in wwwroot that needs to be copied or this line can be removed too?